src/main/resources/config/application-default.yml

```diff
@@ -2,10 +2,19 @@ spring:
     profiles:
         active: dev
     datasource:
-        platform: mysql
+        platform: postgres
         driver-class-name: org.postgresql.Driver
         url: jdbc:postgresql://${db_host:localhost}:${db_port:5432}/${db_name:whereabouts}?currentSchema=public
         username: ${db_username:wa}
         password: ${db_password:wa}
         hikari:
             jdbc-url: jdbc:postgresql://${db_host:localhost}:${db_port:5432}/${db_name:whereabouts}
+server:
+    port : 8085
```

Comment:
This is fine, but for me happens to conflict with glowing-bear, because the value is used in example configuration here: https://github.com/thehyve/puppet-transmart_core#1-configuring-a-node-using-hiera. No need to change it here though.